pybloqs/html.py

```diff
@@ -70,23 +70,28 @@ def append_to(parent, tag, **kwargs):
     return new_tag
 
 
-def js_elem(container=None, script=None):
+def construct_element(container=None, content=None, tag=None, element_type=None):
     """
-    Constructs a Javascript element and appends it to the container.
+    Constructs an element and appends it to the container.
 
     :param container: Container to add the element to.
-    :param script: Javascript code.
+    :param content: String representation of content (e.g. JS or CSS)
+    :param type: E.g. "text/javascript" or "text/css
```

Comment:
`element_type`? also add docstring for `tag`

Reply:
@DominikMChrist ^ Still needs fixing